### PR TITLE
Moved showcase and examples out of Readme, created showcase.md

### PR DESCRIPTION
**Pull request by En3Tho**
Something I should have done long ago :)

I'm not sure how to actually showcase stuff now:

Should Readme have a link on showcase? 
Should at least something be showcased in readme?

Gifs on my apps are incoming

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,20 +50,6 @@ _The Documentation matches the most recent Nuget release from the `main` branch
 descriptors. Most classes are safe for threading.
 * **Reactive Extensions** - Use [reactive extensions](https://github.com/dotnet/reactive) and benefit from increased code readability, and the ability to apply the MVVM pattern and [ReactiveUI](https://www.reactiveui.net/) data bindings. See the [source code](https://github.com/gui-cs/Terminal.Gui/tree/master/ReactiveExample) of a sample app in order to learn how to achieve this.
 
-## Showcase & Examples
```
Comment (tig) [inline README.md:53]: There's no link to showcase.md in readme.md. Please leave this header in, and provide a link.

**Review by tig — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tig) [reply README.md:53]: @En3Tho can you please address this?

Comment (En3Tho) [reply README.md:53]: @tig Thank you for you patience. Please take a look

**Review by tig — APPROVED** [review]
Lovely! Thank you.

**Pull request merged**